docs/user-guide/installation/linux.md

```diff
@@ -107,7 +103,7 @@ IPython 6.2.1 -- An enhanced Interactive Python. Type '?' for help.
 In [1]: import helics
 
 In [2]: helics.helicsGetVersion()
-Out[2]: 'x.x.x (20XX-XX-XX)'
+Out[2]: '3.x.x (20XX-XX-XX)'
```

Comment:
Shouldn't the `Building HELICS with python support` section be removed? We don't have swig python interfaces any more.

Reply:
right I will pull it out